plugins/DuskVerb/tools/tuner/baseDelay_sweep.py

```
@@ -162,16 +169,23 @@ def objective(trial):
-    subprocess.run([
-        str(RENDER), "--vst3", str(VST3),
-        "--program", args.preset,
-        "--param", "Dry/Wet=1.0",
-        "--param", "Bus Mode=On",
-        "--prerun-seconds", "5",
-        "--sustained-pink-seconds", "4",
-        "--output-dir", str(out_root),
-    ], env=env, check=False, capture_output=True)
-    print(f"\nBest render in {out_root}")
+    try:
+        res = subprocess.run([
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)